.env.example

```diff
@@ -3,3 +3,5 @@ NGROK_HOSTNAME=
 NGROK_PORT=bot:8080
 SLACK_TOKEN=xoxb-...
 DB_FILE=db/phpmxbot.db
+ENVIRONMENT=prod
+DEBUG=false
```

Comment:
Non-blocking, Question. I do not know if this is a Symphony thing but, if this `DEBUG` variable is used to control the log output, the name  should be used LOG_LEVEL instead and the values usually are in the form of `info`, `notice`, `warning`, `error`, critical`, etc.

Reply:
I think this it's no for logging it's only how the application show the error pages, for example if this value is true only show the error and that it, in other case if this false, should detail that can help us to develop and find potentially bugs